mne/realtime/epochs.py

```diff
@@ -126,6 +126,14 @@ class RtEpochs(BaseEpochs):
         The events associated with the epochs currently in the queue.
     verbose : bool, str, int, or None
         See above.
+
+    Notes
+    -----
+    Calling `next()` on an `RtEpochs` object (as internally done when iterating
+     over the object) is blocking, i.e., waits for at most `isi_max` seconds
```

Comment:
The inconsistent indentation breaks the rendering:

https://8009-1301584-gh.circle-artifacts.com/0/html/generated/mne.realtime.RtEpochs.html